tools/integration_tests/kernel_list_cache/infinite_kernel_list_cache_test.go

```diff
@@ -527,7 +527,7 @@ func TestInfiniteKernelListCacheTest(t *testing.T) {
 
 	// Define flag set to run the tests.
 	flagsSet := [][]string{
-		{"--kernel-list-cache-ttl-secs=-1"},
+		{"--kernel-list-cache-ttl-secs=-1", "--metadata-cache-ttl-secs=0"},
```

Comment:
Add TODO here , that we can change this with negative-cache-ttl 0 when it will introduce.

Reply:
Added